docs/geojson.md

```diff
@@ -413,17 +424,28 @@ Like with encoding, Spatial-K objects can also be decoded using `kotlinx.seriali
 === "Kotlin"
 
     ```kotlin
-    val feature: Feature = Json.decodeFromString(Feature.serializer(), "{...feature...}")
+    val feature: Feature = GeoJson.decodeFromString(Feature.serializer(), "{...feature...}")
```

Comment:
out of scope for this pr, but we should eventually move such snippets to .kt files and [include them](https://facelessuser.github.io/pymdown-extensions/extensions/snippets/) from there, so they get type checked at build time

Reply:
filed https://github.com/maplibre/spatial-k/issues/174 to track that